### PR TITLE
Updating the location of WP as per new on device version, makes it optio...

### PR DESCRIPTION
**Pull request by ejzn**
...nal in setup

and will bundle resources that are included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ This repo contains the code for the BlackBerry 10 WebWorks Packager and submodul
     - [*Mac/Linux*] `source *BBNDK installation directory*/bbndk-env.sh`
     - [*Windows*] `*BBNDK installation directory*\bbndk-env.bat`
 5. **Webplatform setup:**<br />
-    Copy the following webplatform files from the latest [BlackBerry 10 WebWorks SDK](https://developer.blackberry.com/html5/download/sdk) into `./dependencies/webplatform/framework/clientFiles/`. For more details on how to install go [here](https://developer.blackberry.com/html5/documentation/install_and_configure_ww_bb10_2007535_11.html).
+    ***Optional step!***<br />
+    If you do not wish to use the supplied version of webplatform on device, you may complete the following steps:
```
Comment (nukulb) [inline README.md]: you should remove this step completely, since we don't want anyone doing it.

Comment (nukulb) [inline README.md]: and it won't be a part of the BlackBerry 10 SDK

Comment (ejzn) [inline README.md]: Sure, can do.

File: lib/file-manager.js [hunk 2]
```diff
@@ -168,13 +175,20 @@ function copyWWE(session, target) {
 }
 
 function copyWebplatform(session, target) {
+
     var wpSrc = path.normalize(session.conf.ROOT + "/webplatform.js"),
```
Comment (nukulb) [inline lib/file-manager.js:179]: what does this path evaluate to?

Comment (ejzn) [inline lib/file-manager.js:179]: Evaluates to the root of the framework folder I believe.

Comment (ejzn) [inline lib/file-manager.js:179]: You can override the webplatform.js or the 18n.js by putting it in the root of your framework. From there it is bundled into the root of your application where the platform tag can pick it up.

**Pull request closed without merging**